client/src/modules/shopping-list/index.jsx

```diff
@@ -78,6 +82,26 @@ class ShoppingList extends Component {
     this.hideUpdateForm();
   };
 
+  adjustDialogBox = (listId, isArchived) => {
```

Comment:
I would name it renderDialogBox as it method renders dialogBox conditionally, cause then in render method it's not clear when we see statement like this` {showDialogBox && this.adjustDialogBox(listId, isArchived)}`
This version would be more clear for someone reading render method 
` {showDialogBox && this.renderDialogBox(listId, isArchived)}`

Reply:
You're right, but this method is already removed, because of new component ArchivedList which take over some functionalities